packages/repository-modernizer/src/restructure-config.js

```diff
@@ -531,18 +532,45 @@ function removeUnwantedChars(key, val, filePath) {
         } else if (val.charAt(0) == "[") {
             str = '"' + key + '"' + ":" + "[";
             let tokens = val.substring(1, val.indexOf("]"));
+
             if (tokens.length > 0) {
                 tokens = tokens.split(",");
                 for (let i = 0; i < tokens.length; i++) {
                     if (i == tokens.length - 1) {
-                        str = str + '"' + tokens[i] + '"';
+                        if (
```

Comment:
Can you add a few comments in the different if-else blocks to make the logic easier to understand?

Reply:
done ..